app/src/main/java/com/xmartlabs/gong/data/repository/auth/UserRemoteSource.kt

```diff
@@ -20,7 +20,7 @@ class UserRemoteSource {
   suspend fun signIn(id: String, password: String) = withContext(Dispatchers.IO) {
     @Suppress("MagicNumber")
     delay(100) // Simulate network delay
-    if (id.toLowerCase(Locale.ROOT) == VALID_ID && password == VALID_PASSWORD) {
+    if (id.lowercase(Locale.ROOT) == VALID_ID && password == VALID_PASSWORD) {
```

Comment:
what about using string compare functions with the ignore case parameter in true?